src/components/layout/Footer.tsx

```diff
@@ -109,14 +109,20 @@ const Footer = () => (
 );
 
 const FooterNavbar = styled(Navbar)`
+  .copy-text {
```

Comment:
You can remove these .copy-text class blocks from here and line 123 now, as well